src/main/java/com/student/user/app/controllers/item/AddStudentController.java

```diff
@@ -83,6 +100,15 @@ protected void onAddStudClicked(ActionEvent event) {
             displayError("Invalid input in the following fields:", invalidFields);
             return;
         }
+        try {
```

Comment:
this try catch was not used at all

Reply:
this validation is blocking the user which setting the birthday year in a present year only